cmd/root.go

```diff
@@ -121,12 +120,12 @@ func initConfig() {
 		if configFilePath, exists := os.LookupEnv("INVENTORY_API_CONFIG"); exists {
 			absPath, err := filepath.Abs(configFilePath)
 			if err != nil {
-				log.Fatalf("Failed to resolve absolute path for config file: %v", err)
+				fmt.Printf("Failed to resolve absolute path for config file: %v", err)
```

Comment:
Why change the `log.Fatatf` to `fmt.Printf`?

Reply:
When I was testing out applying the filter to the log messages I found that logs that were made before the filter has been applied caused strange errors to occur, 
An example i found was:
```
make inventory-up
podman logs inventory-api-inventory-api-1
```
Would result in 
```
❯ podman logs inventory-api-inventory-api-1
FATAL msg=Using config file: /inventory-api-compose.yaml
FATAL msg=Using config file: /inventory-api-compose.yaml
FATAL msg=Using config file: /inventory-api-compose.yaml
FATAL msg=Using config file: /inventory-api-compose.yaml
FATAL msg=Using config file: /inventory-api-compose.yaml
FATAL msg=Using config file: /inventory-api-compose.yaml
FATAL msg=Using config file: /inventory-api-compose.yaml
```

And changing the log.Fatalf -> fmt.Printf stopped the errors from occuring.

I attempted to call the logger earlier in the program but that caused other problems like: 
- Not being able to access the log level from the config file because it hasn't been read yet (No filtering anymore)
- Goroutine errors

Reply:
Turns out that I was making an [info](https://github.com/project-kessel/inventory-api/blob/28c7c16393acde038b78639317667b0fdaec40e5/cmd/root.go#L149) message a Fatal message by mistake so this should be fixed now